src/JsonMaker.cpp

```diff
@@ -0,0 +1,74 @@
+/******************************************************/
```

Comment:
It might be worth either moving to `.cpp` files or adding a `.gitignore` entry for the generated files from the build tool. Ideally, we don't want build outputs committed into the repo.

Reply:
Changed extension of .ino files to .cpp files.